en/log/plotjuggler_log_analysis.md

```diff
@@ -1,77 +1,90 @@
+[PlotJuggler](https://github.com/facontidavide/PlotJuggler) can be used to analyze ULogs for in-depth, development purposes.
+It is incredibly useful as every uORB topic is exposed / can be graphed, as well as having custom functions to modify the data (e.g. from Quaternion values to Roll / Pitch / Yaw).
 
 ## Installation
 
 You can find the latest releases of the Plot Juggler [here](https://github.com/facontidavide/PlotJuggler/releases).
 
-### Notes on Windows distribution
+#### Notes on Windows distribution
 
-The latest version of Plot Juggler may not work, in that case you can always fallback to the old one that is known to work with Windows [here](https://github.com/facontidavide/PlotJuggler/releases/tag/2.8.4).
+The latest version of PlotJuggler for Windows may not work.
+In this case fallback to [v2.8.4 here](https://github.com/facontidavide/PlotJuggler/releases/tag/2.8.4) (this is known to work with Windows).
 
-### Notes on AppImage for linux distributions
+#### Notes on AppImage for Linux distributions
 
-When you double-click on the downloaded AppImage, it might have an error like: "Can not open!". For this, you need to change the access settings of the AppImage file like this:
+If the downloaded AppImage does not open, you may need to change its access settings.
+This is done in the terminal using the following command:
 
 ```bash
 chmod 777 <Path-To-PlotJuggler-AppImage>
 ```
 
 ## General Usage
 
-Two most simple / frequently used interface is "Searching" for the logged uORB topic, and "Drag and Dropping" a field in a specific topic onto the graph view, demonstrated in the diagram below.
+Two most common tasks are "Searching" for the logged uORB topic, and "Drag and Dropping" a field in a specific topic onto the graph view.
+These are demonstrated in the diagram below.
 
 ![Plot Juggler basic usage](../../assets/flight_log_analysis/plot_juggler/plotjuggler_timeseries_search_and_drop.svg)
 
-
 ### Splitting Horizontally / Vertically : Multi Panel
 
-One of the most powerful feature is splitting the screen in either horizontal/vertical orientation and having different graphs displayed together with a synchronized time-bar on top, as you move the time cursor in the bottom. This is demonstrated in an animation below:
+One of the most powerful feature is splitting the screen in either horizontal/vertical orientation and displaying different graphs at the same time ( with a synchronized time-bar on top, as you move the time cursor in the bottom).
+
+This is demonstrated in an animation below:
 
-![Plot Juggler Multi panel demonstration](/assets/flight_log_analysis/plot_juggler/plotjuggler_dragdrop_multipanel.gif)
+![Plot Juggler Multi panel demonstration](../../assets/flight_log_analysis/plot_juggler/plotjuggler_dragdrop_multipanel.gif)
```

Comment:
Isn't it better to have the absolute link (/assets) instead of relative (../../assets), so that when we move the location of this doc, the link will still be valid? @hamishwillee

Reply:
It would be desirable if it worked: https://vuepress.vuejs.org/guide/assets.html#relative-urls

But you don't have to worry, I have thought of this. Except for just one doc in the root our structure is such that everything is in a folder /en/folder/doc, so that the relative link always goes back to the root `/assets`